apps/errstr.c

```diff
@@ -52,16 +52,19 @@ int errstr_main(int argc, char **argv)
         }
     }
 
+    /*
+     * We're not really an SSL application so this won't auto-init, but
+     * we're still interested in SSL error strings
+     */
+    OPENSSL_init_ssl(OPENSSL_INIT_LOAD_SSL_STRINGS
+            | OPENSSL_INIT_LOAD_CRYPTO_STRINGS, NULL);
```

Comment:
Nit: align second line with parenthesis

Reply:
fixed alignment.  (Moving it out of the for loop was the key part :)